### ISSUE TITLE
Ignores meta tags with space (BZ#1129)

### ISSUE THREAD

**Issue by tbm** [ledger/ledger#1129]
Note: the issue was created automatically with bugzilla2github

Bugzilla bug ID: BZ#1129
From: Martin Michlmayr (@tbm)

**Comment by tbm** [comment]
I'd say this is invalid as metadate keys may not contain space.

**Comment by jwiegley** [comment]
I agree, this is not valid, it would need to be `Due_Date` instead of `Due Date`.
